cluster-autoscaler/simulator/delta_cluster_snapshot.go

```diff
@@ -353,7 +355,25 @@ func (snapshot *deltaSnapshotNodeLister) Get(nodeName string) (*schedulerframewo
 
 // IsPVCUsedByPods returns if PVC is used by pods
 func (snapshot *deltaSnapshotStorageLister) IsPVCUsedByPods(key string) bool {
-	return false
+	data := snapshot.data
+	if data.pvcNamespaceMap != nil {
+		if data.pvcNamespaceMap[key] > 0 {
```

Comment:
Same as below, can be simplified from `if cond { return true } return false` to just `return cond`.